include/odp/api/spec/ml_types.h

```diff
@@ -329,6 +329,13 @@ typedef struct odp_ml_capability_t {
 
 /** Machine learning configuration parameters */
 typedef struct odp_ml_config_t {
+	/** Engine ID to be configured.
+	 *
+	 * In a system with multiple ML engines, this parameter selects the engine to be configured.
+	 * The default value is 0.
```

Comment:
The description should state which engine_id values are valid, i.e. values less than odp_ml_engine_count().

Reply:
Updated the documentation with the permitted range of values for engine_id.